public/app/features/search/page/components/columns.tsx

```diff
@@ -152,6 +145,13 @@ export const generateColumns = (
       Header: 'Location',
       width,
     });
+    availableWidth -= width;
+  }
+
+  // Show tags if we have any
+  if (access.tags) {
```

Comment:
I was reading through Amy's designs notes and she said:

> X If no tags exist today show no tags - (Amy) no, we’re not doing this

 So this means, we always want to show the tags column, even if we don't have tags